RestApiForGroup2/server.js

```diff
@@ -0,0 +1,199 @@
+const req = require('request');
+const pg = require('pg');
+const express = require('express');
+const router = express.Router();
+const path = require('path');
+const body_parser = require('body-parser');
+const  HttpStatus  =  require('http-status-codes');
+const hostname = '127.0.0.1';
+const port = 3000;
+
+var app = express();
+app.use(express.static(__dirname + '/public'));
+app.use(body_parser.json());
+app.use(body_parser.urlencoded({
+    extended: true
+}));
+
+var cn = {
+    host: 'localhost',
+    port: 5432,
+    database: 'postgres',
+    user: 'postgres',
+    password: '1234',
+};
+
+
+app.post("/entities", function(req, res) {
+    var entity = req.body;
+    var pseudonames = [];
+    for (item of entity.pseudonym) {
+        pseudonames.push(item);
+    }
+    var works = [];
+    for (item of entity.work) {
+        works.push(item);
+    }
+    var releases = [];
+    for (item of entity.release) {
+        releases.push(item);
+    }
+    var srcLink = [];
+    for (item of entity.srcLink) {
+        srcLink.push(item);
+    }
+    console.log(pseudonames);
+    pg.connect(cn, function(err, client, done) {
```

Comment:
I have fixed this comment in :https://github.com/ShilpaGhanashyamGore/StructuredData/commit/a162c2f2c1076b8e41a9c045852631310f486a1e
Please check it.